src/NuGet.Core/NuGet.Packaging/PackageReaderBase.cs

```diff
@@ -359,8 +359,6 @@ protected IEnumerable<FrameworkSpecificGroup> GetFileGroups(string folder)
             {
                 yield return new FrameworkSpecificGroup(framework, groups[framework].OrderBy(e => e, StringComparer.OrdinalIgnoreCase));
             }
-
-            yield break;
```

Comment:
Is this intentional? according to @emgarten `yield break` makes it return empty enumerable rather than `null`